tests/test_outbounds.py

```diff
@@ -0,0 +1,41 @@
+# SPDX-FileCopyrightText: 2021 Jens Erdmann
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import json
+import pytest
+
+from flict.impl import FlictImpl
+from tests.args_mock import ArgsMock
+
+def _test_expression(expression, result):
```

Comment:
Can we move that function to a common class/file? I already see code duplication here

Reply:
Not really because it calls a different method of FlictImpl in each test.